frontend/src/static/css/v_summary.scss

```diff
@@ -2,6 +2,26 @@
 
 /* summary chart */
 #summary,
+.fileTypes {
```

Comment:
The indentation doesn't seems to be right here.

Reply:
If we follow the indentation in other css then means I probably need to shift everything below #summary to right by 2 space though (all other style inside #summary)......Should I do it?